docs/03-extensions/stripe/how-to/apple-pay-google-pay.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Enable Apple Pay and/or Google Pay
+description: "This guides explains how to setup Apple Pay and/or Google Pay"
```

Comment:
**thought:** I don't really feel like this page "explains" anything 😅. Since it's automatically handled by the Stripe extension, it's basically links to Stripe docs.

**suggestion(non-blocking):** Rephrase this. No suggestion comes to my mind really, it's fine if you can't find any, I don't think it's really important.

Reply:
reworded